common/predictive-text/worker/index.ts

```diff
@@ -233,16 +240,22 @@ class LMLayerWorker {
    * @param scope A global scope to install upon.
    */
   static install(scope: DedicatedWorkerGlobalScope): LMLayerWorker {
-    let worker = new LMLayerWorker({ postMessage: scope.postMessage });
+    let worker = new LMLayerWorker({ postMessage: scope.postMessage, importScripts: scope.importScripts.bind(scope) });
     scope.onmessage = worker.onMessage.bind(worker);
 
+    // Ensures that the worker instance is accessible for loaded model scripts.
+    // Assists unit-testing.
+    scope['LMLayerWorker'] = worker;
+    scope['models'] = models;
```

Comment:
TS will report errors if I do the unbracketed form unless `scope`'s type is extended to support the properties.  We're creating these scope-global variables for use by loaded model code that will be compiled JS, not for internal use by the LMLayerWorker itself, so that extension wouldn't yield much if any benefit elsewhere.

Reply:
I'd like to see that reason documented in a code comment for future maintainers please.